pkg/monitortests/network/disruptionserviceloadbalancer/monitortest.go

```diff
@@ -344,3 +356,31 @@ func httpGetNoConnectionPoolTimeout(url string, timeout time.Duration) (*http.Re
 
 	return client.Get(url)
 }
+
+// Uses the first node in the cluster to verify the LoadBalancer host is active before returning
+func checkHostnameReady(tcpService *corev1.Service, nodeTgt string) error {
+	for i := 0; i < 60; i++ {
+		ctx, cancel := context.WithTimeout(context.Background(), 60*time.Second)
+		defer cancel()
+		fmt.Printf("Checking load balancer host is active \n")
+		lbTgt := tcpService.Status.LoadBalancer.Ingress[0].Hostname
+		cmd := exec.CommandContext(ctx, "oc", "debug", nodeTgt, "--", "/bin/bash", "-c", "dig +short "+lbTgt)
```

Comment:
We were discussing something similar in another piece of code earlier today.  Are you familiar with the `oc` wrappers vs exec command?  Could be worth switching just for the sake of future maintainability 

Example: https://github.com/openshift/origin/blob/a1d43cfa1c5d7869f375aa1f8f64a97e5b4ba8cf/test/extended/util/nodes.go#L44

Reply:
Yes, we are - interesting thing in this code, we couldn't get it to run in the context of this test.

Reply:
We can double check.

Reply:
One other thing worth reviewing is the loop / sleep.  [PollUntilContextTimeout](https://pkg.go.dev/k8s.io/apimachinery/pkg/util/wait#PollUntilContextTimeout) would be another consideration.  Looks like worst case scenario this could block for 2 hours?  Understand that isn't likely and doubtful the exec would take a minute to timeout each time.

Reply:
I read that there are cases where LB take over 45 minutes (and in our testing too). So we opted to expand to 60 minutes to give a buffer during activation. The PollUntilContextTimeout could be lowered form our perspective to 10 seconds, we opted to think in the worst of case.

Reply:
My thought was if you used PollUntilContextTimeout you could set your max timeout to 45, 60, whatever you needed.  Then you wouldn't need the loop or sleep.  You could still use the 1 minute context for you exec (or oc call if that works) and configure whatever delay you wanted in between calls, currently 1 minute.

Similar to the cleanup call that has an overall timeout for the main call but then calls a helper method.  
 https://github.com/openshift/origin/blob/a1d43cfa1c5d7869f375aa1f8f64a97e5b4ba8cf/pkg/monitortests/network/disruptionserviceloadbalancer/monitortest.go#L314

Reply:
Running `oc.AsAdmin()` in the latest commit